docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -44,10 +44,14 @@ id: Interview Order
 code: |
   template_upload
   # debug
-  get_all_fields
-  interview.title
+  
+  interview.form_type
   interview_intro_prompt
-  interview.typical_role  
+  interview.title
+  interview.typical_role
+  
+  get_all_fields
+  
```

Comment:
So the order of questions is still confusing to me. Right now it's form information + short title -> action title -> then full title and description. IMO, title, short title, and description -> action title -> form information or title, short title, and description -> form information -> action title feels easier to understand. Asking about the short title before full title is the main thing that seems unnatural.

Reply:
I figured the form information would help the developer get ready for writing a good description for the person using the form. It might not really turn out to be that useful, though. I'm happy to switch it around.

Reply:
That is a good point, I agree that the form info probably helps with writing the description. Maybe it should be form info -> title, short title, and description -> action title.